apps/appclient/mattermost_client.go

```diff
@@ -25,7 +25,7 @@ func AsBot(cc apps.Context) *Client {
 }
 
 func AsActingUser(cc apps.Context) *Client {
-	return as(cc.ActingUserID, cc.ActingUserAccessToken, cc)
+	return as(cc.ActingUser.Id, cc.ActingUserAccessToken, cc)
```

Comment:
What is the rationale for the change here? We'd need to check for `ActingUser != nil`, no?